docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -269,189 +249,253 @@ qc.cx(0,1)
+[`qiskit.primitives.StatevectorSampler`](../api/qiskit/qiskit.primitives.StatevectorSampler). If you want to maintain
+your pre-existing workflow using a Sampler V1 implementation, you can also use the
+[`qiskit.primitives.Sampler`](../api/qiskit/qiskit.primitives.Sampler) class.
 
-```python
-from qiskit.primitives import Sampler
+<Tabs>
+  <TabItem value="SamplerV2" label="StatevectorSampler (V2)">
+    ```python
+from qiskit.primitives import StatevectorSampler
 
-sampler = Sampler()
+sampler = StatevectorSampler()
 ```
+  </TabItem>
 
-### Run and get results
-
-Now that you have defined your `sampler`, run it by calling the [`qiskit.primitives.Sampler.run`](../api/qiskit/qiskit.primitives.Sampler#run) method, which returns an instance of [`qiskit.providers.JobV1`](../api/qiskit/qiskit.providers.JobV1). You can get the results from the job (as a [`qiskit.primitives.SamplerResult`](../api/qiskit/qiskit.primitives.SamplerResult) object) with the [`qiskit.providers.JobV1.result`](../api/qiskit/qiskit.providers.JobV1#result) method.
-
-```python
-job = sampler.run(qc)
-result = job.result()
-print(result)
-```
+  <TabItem value="SamplerV1" label="Sampler (V1)">
+    ```python
+from qiskit.primitives import Sampler
 
-```python
-SamplerResult(quasi_dists=[{0: 0.4999999999999999, 3: 0.4999999999999999}], metadata=[{}])
+sampler = Sampler()
 ```
+  </TabItem>
+</Tabs>
+### Run and get results
 
-While this example only uses one [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit), you can sample multiple circuits by passing a `list` of [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) instances to the [`qiskit.primitives.Sampler.run`](../api/qiskit/qiskit.primitives.Sampler#run) method.
-
-### Get the probability distribution
-
-From these results you can extract the quasi-probability distributions with the attribute [`qiskit.primitives.SamplerResult.quasi_dists`](../api/qiskit/qiskit.primitives.SamplerResult#quasi_dists).
-
-Even though there is only one circuit in this example, [`qiskit.primitives.SamplerResult.quasi_dists`](../api/qiskit/qiskit.primitives.SamplerResult#quasi_dists) returns a list of [`qiskit.result.QuasiDistribution`](../api/qiskit/qiskit.result.QuasiDistribution)s.
-`result.quasi_dists[i]` is the quasi-probability distribution of the `i`<sup>th</sup> circuit.
+<Tabs>
+  <TabItem value="SamplerV2" label="StatevectorSampler (V2)">
+    ```python
+    # Circuits and parameters must obey the Instruction Set Architecture (ISA) of a particular backend.
+    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+
+    pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+    isa_circuit = pm.run(circuit)
+
+    # execute 1 circuit with 3 parameter sets using Sampler V2
+    job = sampler.run([(isa_circuit, [vals1, vals2, vals3])])
+    counts = job.result()[0].data.meas.get_counts()
```

Comment:
We talk about retrieving counts in the section below.

```suggestion
    pub_result = job.result()[0]
```

Reply:
It is kind of messy to add code blocks as code suggestions, but I believe that the output of the print should be:  > Result class: <class 'qiskit.primitives.PubResult'>